db/migrate/20220309152540_create_recipe_foods.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRecipeFoods < ActiveRecord::Migration[7.0]
```

Comment:
there is already a migration file for recipe foods, Kindly check it so we delete and work with just one.